contracts/hardhat.config.ts

```diff
@@ -22,6 +22,28 @@ task("accounts", "Prints the list of accounts", async (_taskArgs, hre) => {
   }
 });
 
+// Don't run this unless you really need to...
+task("privateKeys", "Prints the private keys for accounts")
+  .addParam("force", "Whether the command should be run", false, types.boolean)
+  .setAction(async ({ force }: { force: boolean }, hre) => {
+    if (!force) {
+      throw new Error("are you sure you want to run this task? (--force true)");
+    }
+
+    const separator = "-".repeat(3);
+    console.log(separator);
+
+    for (let i = 0; i < accounts.count; i++) {
+      const wallet = hre.ethers.Wallet.fromMnemonic(
+        accounts.mnemonic,
+        `m/44'/60'/0'/0/${i}`,
+      );
+      console.log(`${i}: ${wallet.address}`);
+      console.log(wallet.privateKey);
+      console.log(separator);
+    }
+  });
```

Comment:
@jzaki This was useful for getting pks for use in local dev, but if you think it might accidentally expose too much I can remove.

Reply:
As long as that's not auto-run :) Good `--force true` requirement :+1: